### PR TITLE
DOCS-5912 update

### PR DESCRIPTION
**Pull request by george-hoffer**
https://www.mulesoft.org/jira/browse/DOCS-5912
Also added edits and added better wraps in the code example.

### CODE REVIEW

File: modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc [hunk 1]
```diff
@@ -86,19 +94,19 @@ timing out. Zero (0) means wait forever.
 . Click Test Connection to ensure that your credentials are accepted at the server endpoint.
```
Comment (valkyrie69) [inline modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc]: UI should be bolded Click **Test Connection**.... click **Enable DataSense**...etc

File: modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc [hunk 2]
```diff
@@ -134,7 +142,7 @@ To build and run this demo project, you need:
 ** dynamics365.clientId - The application ID assigned to your app when you registered it with Azure Active Directory. You can find this in the Azure Portal. Click Active Directory, click the directory, choose the application, and click Configure.
 ** dynamics365.clientSecret - The application secret that you created in the app registration portal for your app. This should not be used in a native app, because a client secret cannot be reliably stored on a device. The client secret is required for web apps and web APIs, which have the ability to store the client secret securely on the server side.
 ** dynamics365.tokenRequestEndpoint - The token endpoint to call to get an access token. +
-Example: `+https://login.windows.net/TENANT_ID/oauth2/token` where TENANT_ID is the Azure Active Directory ID.
+Example: `+https://login.windows.net/TENANT_ID/oauth2/token+` where TENANT_ID is the Azure Active Directory ID.
```
Comment (valkyrie69) [inline modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc:153]: Should it be `TENANT_ID` ?

**Review by valkyrie69 — APPROVED** [review]
Assume local build and link checking was done. Looks good! Minor comments you can address some other time if it's out of scope for this PR.

File: modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc (hunk 1 above)
Comment (george-hoffer) [reply modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc]: Fixed

File: modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc (hunk 2 above)
Comment (george-hoffer) [reply modules/ROOT/pages/microsoft/microsoft-dynamics-365-connector.adoc:153]: TENANT_ID is a placeholder and not an actual value.

**Pull request merged**